test/moonwall.config.json

```diff
@@ -1128,6 +1143,32 @@
                 }
             ]
         },
+        {
+            "name": "chopsticks_tanssi_upgrade",
+            "testFileDir": ["suites/rt-upgrade-chopsticks-orchestrator"],
+            "foundation": {
+                "type": "chopsticks",
+                "rtUpgradePath": "../target/release/wbuild/starlight-runtime/starlight_runtime.compact.compressed.wasm",
+                "launchSpec": [
+                    {
+                        "name": "db",
+                        "type": "parachain",
```

Comment:
Shouldn't this be relaychain or something?

Reply:
We have type - parachain in all the networks, should I change it to relaychain?
This is how they [explained](https://github.com/evgeny-s/moonwall/blob/bfaa04e67e421a918a8a708aeacf3eb88f12bd7e/packages/types/config_schema.json#L52) this param:
```
"type": {
          "description": "An optional type of either \"relaychain\" or \"parachain\".",
          "enum": ["parachain", "relaychain"],
          "type": "string"
        },
```

Reply:
ok, it doesn't affect tests, updated to relaychain in all the relays.
Thanks!